### DISCUSSION TITLE
Can i use LINQ in Pythonnet?

### DISCUSSION THREAD

**Discussion by DynaBimm** [pythonnet/pythonnet#1932] [Q&A] [answered]
Hello!
I have small code in IronPython 2.7 and it works with ParallelLINQ, utilizing all cores of CPU.
In host app IronPython was changed to CPython 3.8. And i can't use LINQ anymore.
Can i connect LINQ to CPython 3.8 with Pythonnet (maybe alpha version works).

Thanks in advance!

**Comment by filmor** [comment, marked as answer]
Proper LINQ support requires https://github.com/pythonnet/pythonnet/issues/55 to be implemented.
